apps/framework-cli/src/cli/routines/migrate.rs

```diff
@@ -218,7 +218,11 @@ fn validate_table_databases(
             SerializableOlapOperation::CreateTable { table } => {
                 validate_db(&table.database, &table.name);
             }
-            SerializableOlapOperation::DropTable { table, database } => {
+            SerializableOlapOperation::DropTable {
+                table,
+                database,
+                cluster_name: _,
+            } => {
                 validate_db(database, table);
```

Comment:
How come this is not used here?

Reply:
Good catch. I had validation during plan generation, but this is during plan execution and the plan files could be modified in between. Added the validation here as well.